frontend/test/specs/eventDialogCtrlSpec.js

```diff
@@ -267,19 +267,41 @@
             expect(controller.getStep(1)).toEqual(true);
         });
 
-        it('should not pass from second step', function() {
+        it('should not pass from first step', function() {
             controller.event.title = "";
             controller.nextStep();
             expect(controller.steps).toEqual([true, false, false]);
         });
 
-        it('should not pass from second step', function() {
+        it('should not pass from first step', function() {
             controller.event.title = "Title";
             controller.event.local = "";
             controller.nextStep();
             expect(controller.steps).toEqual([true, false, false]);
         });
-    }); 
+
+        it('should not pass from first step', function() {
+          controller.event.title = "Title";
+          controller.event.address = address;
+          controller.event.local = "Local";
+          controller.event.start_time = undefined;
+          controller.event.end_time = undefined;
+          controller.nextStep();
+          expect(controller.steps).toEqual([true, false, false]);
+        });
+
+        it('should pass from first and second step', function() {
```

Comment:
You should create a test that covers the problem spoken above. In your new logic 
```
controller.event.start_time = undefined;
controller.event.end_time = date_now;
```

Will work, but shouldn't.